guides/common/modules/con_limiting-host-resources.adoc

```diff
@@ -0,0 +1,20 @@
+[id="limiting-host-resources"]
+= Limiting host resources
+
+You can use the Resource Quota plugin to limit users' access to host resources during host provisioning.
+
+In a typical usage scenario, {Project} manages multiple compute resources for multiple departments.
+To share managed resources in a fair and predictable way, administrators can assign resource quotas to users and user groups.
+
+// ordered like shown in the WebUI
+Supported resource types::
+* CPU cores {endash} limit the number of CPU cores
+* Memory {endash} limit the amount of memory
+* Disk space {endash} limit the amount of disk space
```

Comment:
I know I'm late to the party :) But this part seems kind of repetitive to me. 
```suggestion
It is supported to limit the following resource types::
* CPU cores 
* Memory
* Disk space
```

Reply:
Alternative suggestion:
```suggestion
Supported resource types:: You can limit the following host resources:
* CPU cores
* Memory
* Disk space
```